package.json

```diff
@@ -101,5 +105,11 @@
     "typings/package.json.d.ts",
     "bin/rollup",
     "README.md"
-  ]
+  ],
+  "lint-staged": {
```

Comment:
Hey, Prettier maintainer here. Awesome stuff. Just wanted to drop a line and ask if you've heard of [`pretty-quick`](https://github.com/azz/pretty-quick)? It makes this config a little bit cleaner.

Reply:
Hi, thanks for the tip, looks cool!